external/error_report/error_report.c

```diff
@@ -80,7 +81,16 @@ err_status_t error_report_deinit(void)
 error_data_t *error_report_data_create(int module_id, int error_code, uint32_t pc_value)
 {
 	struct timeval ts;
-	gettimeofday(&ts, NULL);
+	struct timespec tv;
+	int retval = clock_gettime(CLOCK_MONOTONIC, &tv);
```

Comment:
When CONFIG_CLOCK_MONOTONIC is enabled, you can use this mode. Did you make a dependency between err_report and clock_monotonic? If you didn't, you should do it.

Reply:
Added a check for MONOTONIC before use. Thanks for spotting this.